llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -1246,6 +1246,17 @@ void DwarfDebug::finishSubprogramDefinitions() {
   }
 }
 
+/// Finalizes DWARF acceleration tables.
+/// Currently it converts DIE entries to offsets in .debu_names entry.
+static void
+finalizeDWARF5AccelerationTables(DWARF5AccelTable &AccelDebugNames) {
+  for (auto &Entry : AccelDebugNames.getEntries()) {
+    for (AccelTableData *Value : Entry.second.Values) {
+      static_cast<DWARF5AccelTableData *>(Value)->normalizeDIEToOffset();
+    }
+  }
+}
+
```

Comment:
Perhaps this function could move to be a member of `DWARF5AccelTable`? (& won't need to add a `getEntries` function)